topk-rs/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "topk-rs"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
is this how we do it?

Reply:
we're not publishing topk-rs yet so yeah this is fine, doesn't have any effect now